api/GPU.json

```diff
@@ -14,7 +14,9 @@
           },
           "edge": "mirror",
           "firefox": {
-            "version_added": false
+            "version_added": "preview",
+            "partial_implementation": true,
+            "notes": "Currently supported on Linux and Windows only. See <a href='https://bugzil.la/1616739'>bug 1616739</a>."
           },
           "firefox_android": "mirror",
```

Comment:
Android needs version_false and shouldn't mirror.

Reply:
Done everywhere except where there is an addition flag which means this is disabled by default anyway.